Tests/UnitTests/Plugins/Geant4/Geant4DetectorSurfaceFactoryTests.cpp

```diff
@@ -94,4 +97,97 @@ BOOST_AUTO_TEST_CASE(Geant4DetecturSurfaceFactory_Cylinder) {
   BOOST_CHECK_EQUAL(surface->type(), Acts::Surface::SurfaceType::Cylinder);
 }
 
+BOOST_AUTO_TEST_CASE(Geant4DetecturSurfaceFactory_Transforms) {
+  Acts::GeometryContext gctx;
+
+  G4Box* worldS = new G4Box("world", 1000, 1000, 1000);
```

Comment:
since this is a unit test it is not so important but the `new` statements without `delete` look a bit alarming